MobileCenter/MobileCenter/Internals/Channel/MSChannelDefault.m

```diff
@@ -123,6 +127,9 @@ - (void)flushQueue {
       self.availableBatchFromStorage = YES;
       self.itemsCount = 0;
     }
+
+    // Perhaps notify done flushing.
+    [self perhapsNotifyDoneFlushing];
```

Comment:
This just reads weird, maybe `notifyDoneFlushingIfApplicable`? Hm...I'm too picky,  am I?!

Reply:
I kinda liked it but I am opened to suggestions

Reply:
I'm okay if you like it that way.